bats/tests/registry/creds.bats

```diff
@@ -219,3 +240,11 @@ EOF
     run bash -c "echo \"$REGISTRY\" | \"$CRED_HELPER\" get"
     refute_output --partial '"Secret":"password"'
 }
+
+@test 'verify the docker desktop setting is replaced' {
```

Comment:
The title wasn't immediately clear to me, maybe change to something like:

```suggestion
@test 'verify docker-credential-desktop is replaced with default setting' {
```

Reply:
sure